src/main/views/terms-and-conditions.njk

```diff
@@ -0,0 +1,187 @@
+{% extends "layout.njk" %}
+{% set heading = t('Terms and conditions') %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('By using this service you’re agreeing to the following terms of use.') }}</p>
+
+      <p> {{ t('This includes the <a href="{{  privacyPolicy }}">privacy policy</a>.',
+        { privacyPolicy: 'privacy-policy' }) | safe }}
```

Comment:
link?

Reply:
Internal like referencing
```
static readonly privacyPolicyPage = new RoutablePath('/privacy-policy', false)
```

Reply:
I don't think it works like that,
if you are using a variable then you should be doing:
```
{ privacyPolicy: privacy-policy }
```
what's going in atm is:
```
<a href="privacyPolicy"
```
which would work but not what you want because you want the slash to be at the start

Reply:
@geek-soft

Reply:
@jacmurphy you would need something like:

```
<p> {{ t('This includes the <a href="{{ uri }}">privacy policy</a>.', { uri: ClaimPaths.privacyPolicyPage.uri() }) | safe }}
```